remote-api/src/test/java/org/alfresco/rest/api/impl/CategoriesImplTest.java

```diff
@@ -479,6 +503,37 @@ public void testCreateCategory_includeCount()
             .isEqualTo(0);
     }
 
+    @Test
+    public void testCreateCategory_includePath()
+    {
+        final QName categoryQName = createCmQNameOf(CATEGORY_NAME);
+        final NodeRef categoryNodeRef = createNodeRefWithId(CATEGORY_ID);
+        final NodeRef parentCategoryNodeRef = createNodeRefWithId(PARENT_ID);
+        final ChildAssociationRef parentAssociation = createAssociationOf(parentCategoryNodeRef, CATEGORY_NODE_REF, categoryQName);
+        final Path mockPath = new Path();
+        given(nodesMock.validateNode(PARENT_ID)).willReturn(parentCategoryNodeRef);
+        given(categoryServiceMock.createCategory(parentCategoryNodeRef, CATEGORY_NAME)).willReturn(categoryNodeRef);
+        given(nodesMock.getNode(any())).willReturn(createNode());
+        given(nodeServiceMock.getPrimaryParent(any())).willReturn(parentAssociation);
+        given(parametersMock.getInclude()).willReturn(List.of(INCLUDE_PATH_PARAM));
+        given(nodeServiceMock.getPath(any())).willReturn(mockPath);
+        final List<Category> categoryModels = new ArrayList<>(prepareCategories());
+
+        // when
+        final List<Category> actualCreatedCategories = objectUnderTest.createSubcategories(PARENT_ID, categoryModels, parametersMock);
+
+        then(categoryServiceMock).should().createCategory(any(), any());
+        then(categoryServiceMock).shouldHaveNoMoreInteractions();
+
+        assertThat(actualCreatedCategories)
+                .isNotNull()
+                .hasSize(1)
+                .element(0)
+                .extracting(Category::getPath)
+                .isNotNull()
+                .isEqualTo("");
```

Comment:
I don't know if this is simple enough to achieve but it would be better to connect this assertion to `mockPath` variable instead of using empty String (`""`) comparison in here and all other similar assertions you added.

Reply:
Right refactored